spec/models/user_building_spec.rb

```diff
@@ -1,4 +1,13 @@
 require 'rails_helper'
 
 describe UserBuilding do
+	it { should have_many :units }
+	it 'is invalid without an address' do
```

Comment:
This can be condensed to a one-liner using shoulda matcher https://github.com/thoughtbot/shoulda-matchers. Take a look at the examples like `it { should validate_presence_of(:name) }` Let me know if that doesn't work. Additional shoulda-matchers configuration may need to happen.